scripts/hf_example.py

```diff
@@ -0,0 +1,109 @@
+import argparse
```

Comment:
This should be named something more descriptive, that includes the fact it is doing some comparison in performance between hf and fms as well as being an example.

Reply:
hmm.. this code does include performance output, but the purpose is more to highlight how HF weights can be used with our models + compile in an HF API. The benchmarking code is more to validate that we actually have it working/compiling correctly. maybe you'd prefer `hf_compile_example.py` ?

For checking performance characteristics I'd favor using the benchmark_inference.py script directly and use this only to show how to compile with HF. Hopefully we can also simplify this script, since it's currently a bit tricky to get an HF adapted model to compile correctly.

Reply:
Yes, I think `hf_compile_example.py` would be better